components/layout_thread/lib.rs

```diff
@@ -875,7 +876,8 @@ impl LayoutThread {
                         root_flow.overflow.scroll.size
                     };
 
-                    let origin = Rect::new(Point2D::new(Au(0), Au(0)), root_size).to_layout();
+                    let origin =
+                        Rect::new(Point2D::new(Au::zero(), Au::zero()), root_size).to_layout();
```

Comment:
I guess this could use `Point2D::default()`

Reply:
`Point2D::zero()` or `Point2D::origin()` also seem to work. I think I will go with `Point2D::zero()` because it's more obvious what it does.